packages/components/close-button/src/CloseButton.styles.tsx

```diff
@@ -0,0 +1,34 @@
+import { cva } from 'class-variance-authority'
+
+export const closeButtonStyles = cva(
+  ['flex h-full items-center justify-center focus-visible:outline-none'],
+  {
+    variants: {
+      cursor: {
+        pointer: ['cursor-pointer'],
+        disabled: ['cursor-not-allowed'],
+      },
+    },
+    defaultVariants: {
+      cursor: 'pointer',
+    },
+  }
+)
+
+export const closeButtonIconStyles = cva(
+  [
+    'inline-flex rounded-full',
+    'focus-visible:ring-outline-high ring-inset focus-visible:outline-none focus-visible:ring-2',
```

Comment:
I think for the Popover the styles are different. I am not sure we should have a dedicated package for a CloseButton at this stage, maybe it could be part of `Input` or `TextField` to be re-used by TextAreaField and later on we could deep dive into this component ?

I think for Drawer/Dialog it will be yet another design.

Reply:
the chip and the input had the same one and will be very common

https://mantine.dev/core/close-button/